defaults/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+py_version: 2.7.9 # same as AWS
```

Comment:
2.7.9 was the latest I think when the role was first written :smile_cat: This variable is only used when we are compiling our own on the vagrant boxes I think. We're only moving the roles into their own repos so I believe we're safe for the moment and we can create a story to update this and test for breakages.